scripts/lint.sh

```diff
@@ -1,7 +1,10 @@
 #!/bin/bash
-
 set -ex # fail on first error, print commands
 
+echo "Checking code style with black...."
+python -m black bambi --line-length=100  --target-version=py37 --exclude=tests/ --check
```

Comment:
I would suggest adding a pyproject.toml file. Otherwise what will happen is someone will run black locally but not know the line length and black will then fail in CI

https://github.com/arviz-devs/arviz/blob/master/pyproject.toml

Reply:
Nice! Didn't know about that. So many different ways to configure python tooling haha. 

Will add this now, thank you :)

Reply:
Np! This is what code reviews are for.

I personally would include tests in the check because other people have to read them. Its like writing a book where half the code is formatted nicely

and 
the
   other
half is
            formatted
terribly

Why not just format everything that has to be read consistently?
Not required but that's just my opinion

Reply:
4b25a2a

Reply:
Gah! I didn't see you other comment until now. Happy to include tests

Reply:
4cf381c